php/getPhysicalAddress.php

```diff
@@ -1,31 +1,26 @@
 <?php
 
-if($_POST['yes']){
-
     $servername = 'localhost';
     $user = 'root';
     $pass = '';
     $db = 'cs3320';
-    
+
     $con = new mysqli($servername, $user, $pass, $db) or die("Unable to connect");
     if ($con->connect_error) {
         die("Connection failed: " . $con->connect_error);
     }
-    $sql="SELECT * FROM userinformation WHERE '$_POST[userID]'";
+
+    $last_id = $con->insert_id;
```

Comment:
Can you tell me how this works?

Reply:
probably should have deleted line 13 since I don't actually use it & it isn't correct. $con->mysqli_insert_id is a function used to get the last ID auto-generated by a query. 
[Documentation Here.](https://www.php.net/manual/en/mysqli.insert-id.php) We might be able to use that in the future for retrieving address info/order info from the database instead of a cookie?